### DISCUSSION TITLE
Krisp - Zed audio channels integration

### DISCUSSION THREAD

**Discussion by Moshyfawn** [zed-industries/zed#7468] [Feature Requests]
Keyboard clacking is unavoidable when working with your fellow developers over Zed with your microphones open. Discord has solved this by integrating with Krisp, which has [its own SDK](https://krisp.ai/developers/). I wonder if Zed will consider doing something similar.

**Comment by Igonato** [comment]
I'd like to vote nay on Krisp. It's too proprietary. I think in the near future there will be a good open source alternative, if there isn't one already

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
